coalib/results/Diff.py

```diff
@@ -276,23 +284,39 @@ def split_diff(self, distance=1):
         # always.
         yield this_diff
 
-    def range(self, filename):
+    def range(self, filename=None):
         """
         Calculates a SourceRange spanning over the whole Diff. If something is
         added after the 0th line (i.e. before the first line) the first line
         will be included in the SourceRange.
 
         The range of an empty diff will only affect the filename:
 
-        >>> range = Diff([]).range("file")
-        >>> range.file is None
-        False
+        >>> range = Diff([], filename="/file").range()
+        >>> range.file
+        '/file'
```

Comment:
ok my fault, this isn't possible because it's a path and this test fails on windows :/